### PR TITLE
🐞 Fixed all my presences

### PR DESCRIPTION
**Pull request by ACertainCoder**
- IFTTT
- KTANE
- My Nintendo
- Radio.net
- RadioMe
- Rythm

### CODE REVIEW

**Comment by ACertainCoder** [comment]
😂 thanks man. DeepScan isn't working for me. That should be the same issue everywhere.

**Comment by ACertainCoder** [comment]
Seems like there is something still not right. I'll let the website load while I'm not at my pc.

**Comment by doomlerd** [comment]
provide screenshots with all presences working in your edits

**Comment by ACertainCoder** [comment]
IFTTT

KTANE

My Nintendo

Radio.net

RadioMe

Rythm


Here you go.

**Comment by ACertainCoder** [comment]
@doomlerd could you please send me some instructions on how to fix these DeepScan issues. I'm still not able to open their website (neither browser, nor VSCode) and the DeepScan extension in VSCode does not show up any issues.

**Comment by doomlerd** [comment]
image

**Comment by ACertainCoder** [comment]
What the hell? Those aren't even my files!?

**Comment by ACertainCoder** [comment]
😂 I'm not the creator of the Twitch presence

**Comment by doomlerd** [comment]
Update your fork.

**Comment by ACertainCoder** [comment]
Done so, finally no issues. Thank you for your help.

File: My Nintendo/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "author": {
+    "name": "ACertainCoder",
+    "id": "320507572455079949"
+  },
+  "service": "My Nintendo",
+  "description": {
+    "en": "Thanks to My Nintendo makes playing software and interacting with Nintendo even more fun. You can earn points and get rewards such as discounts and additional content for games.",
+    "de": "Dank My Nintendo macht das Spielen von Software und die Interaktion mit Nintendo noch mehr Spaß. Du kannst Punkte sammeln und Belohnungen wie Rabatte und Zusatzinhalte für Spiele erhalten."
+  },
+  "url": ["my.nintendo.com"],
```
Comment (Bas950) [inline My Nintendo/dist/metadata.json:11]: url shouldnt be in an array if its only 1

Comment (ACertainCoder) [reply My Nintendo/dist/metadata.json]: Done 👍

**Pull request merged**